### PR TITLE
used tween in credits

### PR DESCRIPTION
**Pull request by vidalmelero**
What's new:

Credits page animated with tweening engine

### CODE REVIEW

**Comment by luismoyano** [comment]
Good work!

File: Scripts/src/Script/CreditsMenuController.cpp [hunk 1]
```diff
@@ -9,7 +9,7 @@
 #include "Main/GameObject.h"
 #include "Module/ModuleInput.h"
 #include "Module/ModuleScene.h"
-
+#include "Module/ModuleAnimation.h"
```
Comment (marcorod94) [inline Scripts/src/Script/CreditsMenuController.cpp:12]: Code style #include should be in alphabetical order

File: Scripts/src/Script/CreditsMenuController.cpp [hunk 2]
```diff
@@ -33,13 +33,20 @@ CreditsMenuController::CreditsMenuController()
...
 	audio_source = (ComponentAudioSource*)audio_controller->GetComponent(Component::ComponentType::AUDIO_SOURCE);
 
 	GameObject* input_go = App->scene->GetGameObjectByName("Input Manager");
 	const ComponentScript* input_component = input_go->GetComponentScript("InputManager");
 	input_manager = (InputManager*)input_component->script;
+	
+	bg = App->scene->GetGameObjectByName("Credits_bg");
+	anim = App->scene->GetGameObjectByName("Credit_anim");
+	anim2 = App->scene->GetGameObjectByName("Credit_anim2");
+
```
Comment (marcorod94) [inline Scripts/src/Script/CreditsMenuController.cpp:49]: Code Style use more descriptive names for variables

File: Scripts/src/Script/CreditsMenuController.cpp [hunk 3]
```diff
@@ -73,15 +80,41 @@ void CreditsMenuController::Open()
...
 	credits_panel->SetEnabled(false);
 	//audio_source->PlayEvent("Click_backward");
 	audio_source->PlayEvent("menu_select");
+	sequence->Stop();
+	transform->SetTranslation(float3(-1000.0f, 0.0f, 0.0f));
+	anim_transform->SetTranslation(float3(-173.632f, 218.385f, 0.0f));
+	anim_transform2->SetTranslation(float3(501.175f, 218.385f, 0.0f));
+	anim_transform->SetScale(float3(1.0f, 1.0f, 1.0f));
+	anim_transform2->SetScale(float3(1.0f, 1.0f, 1.0f));
+
```
Comment (marcorod94) [inline Scripts/src/Script/CreditsMenuController.cpp:114]: should these numbers be like this? Isn't it better to use dynamic values for the screen width and height?

File: Scripts/src/Script/CreditsMenuController.h [hunk 4]
```diff
@@ -2,6 +2,9 @@
 #define  __CREDITSMENUCONTROLLER_H__
 
 #include "Script.h"
+#include "Animation/Tween/Tween.h"
+#include "Animation/Tween/TweenSequence.h"
+#include "Animation/Tween/LOTween.h"
```
Comment (marcorod94) [inline Scripts/src/Script/CreditsMenuController.h:7]: Cody style

File: Scripts/src/Script/CreditsMenuController.h [hunk 5]
```diff
@@ -28,7 +31,13 @@ class CreditsMenuController : public Script
 
 private:
 	GameObject* credits_panel = nullptr;
+	GameObject* bg = nullptr;
+	GameObject* anim = nullptr;
+	GameObject* anim2 = nullptr;
```
Comment (marcorod94) [inline Scripts/src/Script/CreditsMenuController.h:36]: Same as above

**Pull request merged**